chain/types/percent.go

```diff
@@ -0,0 +1,33 @@
+package types
+
+import (
+	"fmt"
+	"math"
+	"strconv"
+
+	"golang.org/x/xerrors"
+)
+
+// Percent stores a signed percentage as an int64. When converted to a string (or json), it's stored
+// as a decimal with two places (e.g., 100% -> 1.00).
+type Percent int64
+
+func (p Percent) String() string {
+	return fmt.Sprintf(`%d.%d`, p/100, p%100)
```

Comment:
Nit: will not be correct for negative numbers

Reply:
Done. Thanks.